todoapp-main/src/main/java/ru/todoapp/repository/TaskRepository.java

```diff
@@ -0,0 +1,42 @@
+package ru.todoapp.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.simple.JdbcClient;
+import org.springframework.stereotype.Repository;
+import ru.todoapp.model.GetTaskRequestEntity;
+import ru.todoapp.model.SaveTaskEntity;
+import ru.todoapp.model.TaskEntity;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Repository for any task related actions (addition and fetching for now)
+ */
+@Repository
+@RequiredArgsConstructor
+public class TaskRepository {
+    public static final String SAVE_NEW_TASK = "INSERT INTO tda_task (description, datetime, user_uuid) VALUES (?,?,?)";
+
+    public static final String GET_TASKS_FROM_TO = "SELECT id, description, datetime FROM tda_task WHERE user_uuid = ? AND datetime BETWEEN ? AND ?";
+
+    private final JdbcClient jdbcClient;
+
+    /**
+     * Saves new task to DataBase
+     */
+    public void saveNewTask(SaveTaskEntity saveTaskEntity) {
+        var params = Arrays.asList(
+                saveTaskEntity.description(),
+                saveTaskEntity.datetime(),
+                saveTaskEntity.userUUID());
+        jdbcClient.sql(SAVE_NEW_TASK).params(params).update();
+    }
+
+    public List<TaskEntity> getTasksList(GetTaskRequestEntity getTaskRequestEntity) {
```

Comment:
you don't need to use a class parameter in this method. You can just pass user UUID, begin and end date as a parameters, so the class GetTaskRequestEntity become redundant

Reply:
Ok